src/renderer/components/wallet/MnemonicPhrase.tsx

```diff
@@ -13,23 +14,27 @@ type Props = {
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const MnemonicPhrase = React.forwardRef<any, Props>(({ words, onWordClick, readOnly }, ref) => {
   return (
-    <Row ref={ref}>
+    <Styled.Row ref={ref}>
       <Col span={24}>
-        <Card bodyStyle={{ padding: '6px', minHeight: '100px' }}>
-          <div>
+        <Styled.Card bodyStyle={{ padding: '6px', minHeight: '100px' }}>
+          <Row>
             {words.map((word) => (
-              <Button
-                key={word._id}
-                disabled={!!readOnly}
-                danger={word.error}
-                onClick={() => word._id && onWordClick && onWordClick(word._id)}
-                style={{ margin: '6px' }}>
-                {word.text}
-              </Button>
+              <Col sm={{ span: 12 }} md={{ span: 4 }} key={word._id} style={{ padding: '6px' }}>
+                <Styled.Button
+                  type={'text'}
+                  size={'large'}
+                  tabIndex={readOnly ? -1 : 0}
+                  danger={word.error}
+                  readOnly={readOnly}
+                  onClick={readOnly ? undefined : () => word._id && onWordClick && onWordClick(word._id)}
```

Comment:
[nitpick] Do we really need a check of `word._id`? Also I would suggest to provide a default (empty) `onWordClick` while destructuring `props` in this component, so that we have:
```
onClick={readOnly ? undefined : () => onWordClick(word._id)}
```